addons/barcodes_gs1_nomenclature/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Barcode - GS1 Nomenclature',
+    'version': '1.0',
+    'category': 'Hidden',
+    'summary': 'Parse barcodes according to the GS1-128 specifications',
+    'depends': ['barcodes', 'stock'],
```

Comment:
should only depends of barcodes and uom. Would be great to be able to use it without stock